src/platform/site-wide/side-nav/index.js

```diff
@@ -2,14 +2,14 @@
 import React from 'react';
 // Relative
 import startReactApp from '../../startup/react';
-import SideNav from './components/SideNav';
+// import SideNav from './components/SideNav';
```

Comment:
Was this import intentionally commented out, or can it be removed?

Reply:
My mistake, I have removed it.